components/dropdown/dropdown-content-mixin.js

```diff
@@ -93,6 +93,11 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 			},
 			_contentOverflow: {
 				type: Boolean
+			},
+			_dropdownContent: {

```

Comment:
Nit: normally I would try to keep the properties order alphabetically.
```
static get properties() {
	return {
		// public ordered alpha
		...
		// private order alpha
		...
	};
}
```

It might be just me, but I tend to put the property def'n on one line to save some scrolling. That being said, I might have been the source of the original mixup.  🤷‍♂